seqr/views/utils/pedigree_info_utils.py

```diff
@@ -211,6 +212,9 @@ def _parse_header_columns(header):
             column = JsonConstants.FAMILY_NOTES_COLUMN
         elif key.startswith("notes"):
             column = JsonConstants.NOTES_COLUMN
+        elif 'indiv' in key and 'previous' in key:
+            if allow_id_update:
```

Comment:
what's the use case for updating an individual id?

Reply:
Usually, its that someone on the PM team had a typo in their sample manifest so the ID in seqr is wrong. It doesn't happen in the AnVIL case, because we validate their pedigree against the VCF they provide so the IDs have to match. But for internal projects they get created first and loaded later, so you can find out at load time that the ID in seqr is wrong

Also, for RGP we create IDs like RGP_#_1, RGP_#_2 but if we later learn we are not going to get a samples for a parent they will change the name to RGP_#_mom or RGP_#_dad